3rdparty/README_SYCL.md

```diff
@@ -27,19 +28,19 @@ referring to "DPC++".
    conda create -n sycl python=3.8
    conda activate sycl
    ```
-3. Check your environment
+4. Check your environment
    ```bash
    which icx        # /opt/intel/oneapi/compiler/<version>/linux/bin/icx
    which icpx       # /opt/intel/oneapi/compiler/<version>/linux/bin/icpx
    which python     # ${HOME}/.conda/envs/sycl/bin/python
    python --version # Python 3.8.12 :: Intel Corporation
    ```
-4. Config and build
+5. Config and build
    ```bash
    cmake -DBUILD_SYCL_MODULE=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx ..
```

Comment:
we may add `-DBUILD_UNIT_TESTS=ON`, as it is required to build tests (used for demo).

Reply:
Addeded.